src/busctl/busctl.c

```diff
@@ -1500,7 +1500,12 @@ static int message_append_cmdline(sd_bus_message *m, const char *signature, char
 
                         {
                                 unsigned i;
-                                char s[k + 1];
+                                _mallocable_(char) *s;
```

Comment:
I'd prefer to call this `_cleanup_newmalloca_`. All our cleanup funtions start with `_cleanup_`, so `_mallocable_` looks like something completely different.